docs/nconnect-nConnect FAQ.md

```diff
@@ -0,0 +1,62 @@
+---
+id: nconnect-faq
+title: nConnect FAQ
+---
+
+## Is my data secure with nConnect?
+
+nConnect guarantees secure remote access for you from anywhere at any time. Powered by NKN’s global meshed network architecture and unique NKN address, nConnect creates a secure and reliable tunnel enabling all data to be encrypted end-to-end and hop by hop, eliminating the need to go through any centralized cloud server, and never needing a public IP address or an open port.
+
+## Is your service reliable?
+
+Yes. We are extra reliable and production-proven. We are the world’s largest blockchain network in full consensus nodes. NKN’s network of up to 180,000 servers in over 60 countries and regions ensures that high-quality nConnect tunneling nodes are always available.
+
+## Is nConnect a good choice if I care a lot about transfer speed?
+
+Yes. From both our customer’s reports as well as our own internal testing, nConnect consistently meets or beats competitors’ products in terms of transfer speed. Often nConnect can fully utilize users’ maximal uplink capacity.
+
+As a premium paid service, you can choose the highest quality nConnect tunneling servers for high speed and low latency.
+
+## Can I use nConnect on multiple devices?
+
+Yes. nConnect allows you to access all files and Apps on all of your home/office servers and devices through one unified secure connection.
+
+## Do you have a video tutorial for the nConnect setup?
+
+Yes, you can find video tutorials here:
+
+[Synology NAS](https://forum.nkn.org/t/nconnect-user-manual-video-synology-nconnect/2457)
+
+[ASUS NAS](https://forum.nkn.org/t/nconnect-user-manual-video-asustor-nconnect/2501)
+
+[QNAP NAS](https://forum.nkn.org/t/nconnect-user-manual-video-qnap-nconnect/2554)
+
+## How much does nConnect cost?
+
+nConnect is cost-effective.
+
+Pricing starts at $3.99 or 25 RMB per month for up to 40GB.
```

Comment:
nConnect can be used in free mode. Payment for faster data is optional.
And instead of putting those info here, I think it's better to point to its homepage for latest info.

Reply:
Yes, modified.